collections/_article/dev-snapshot-godot-4-3-beta-1.md

```diff
@@ -0,0 +1,255 @@
+
+In terms of bug fixes, the GDScript team fixed lambda hot-reloading ([GH-86569](https://github.com/godotengine/godot/pull/86569)) and fixed out of date errors in depended scripts ([GH-90601](https://github.com/godotengine/godot/pull/90601)). Now GDScript cache will be your friend without restarting the editor!
+
+A lot of efforts have been made to fix GDScript autompletion ([GH-86667](https://github.com/godotengine/godot/pull/86667), [GH-86554](https://github.com/godotengine/godot/pull/86554), [GH-86111](https://github.com/godotengine/godot/pull/86111), and more). For this, we even introduced a testing system to test for autocompletion regressions! ([GH-86973](https://github.com/godotengine/godot/pull/86973))
+
+Finally, to name a few more, we fixed `@warning_ignore` annotation issues ([GH-83037](https://github.com/godotengine/godot/pull/83037)) and adjusted some warnings ([GH-92027](https://github.com/godotengine/godot/pull/92027), [GH-90794](https://github.com/godotengine/godot/pull/90794), [GH-90756](https://github.com/godotengine/godot/pull/90756), and [GH-90442](https://github.com/godotengine/godot/pull/90442)).
+
+### XR
+- Godot 4.3 requires a new version of the vendor's plugin to unlock all platform specific functionality.
+- Fixed tracking issues in the native mobile interface ([GH-91305](https://github.com/godotengine/godot/pull/91305)).
+- Improvements in the core OpenXR loop ([GH-89734](https://github.com/godotengine/godot/pull/89734)), this improves tracking stability and prepares XR for running render processes on a separate thread.
+- Added support for composition layers ([GH-89880](https://github.com/godotengine/godot/pull/89880)), this allows 2D viewports to be displayed at much higher quality within an XR environment.
+- Improvements to foveated rendering/VRS, passthrough, and various other supporting systems.
+- Standardized and enhanced support for Hand/finger tracking, body tracking and face tracking.
+- Support for various Meta extensions such as hand models, render models and scene discovery.
+
+### Documentation
+
+- Add self links to methods, properties, etc. in class reference. For example, this allows you to link directly to a method while reading the description instead of having to scroll up and link the method from the table of methods. ([GH-91537](https://github.com/godotengine/godot/pull/91537)).
```

Comment:
```suggestion
- Add self links to methods, properties, etc. in the online class reference. For example, this allows you to link directly to a method while reading the description instead of having to scroll up and link the method from the table of methods. ([GH-91537](https://github.com/godotengine/godot/pull/91537)).
- Add syntax highlighting and a copy button for code blocks in the built-in documentation viewer. ([GH-89263](https://github.com/godotengine/godot/pull/89263), [GH-87363](https://github.com/godotengine/godot/pull/87363)).
- Add support for deprecated/experimental messages. C# also uses the deprecation message when generating bindings. ([GH-81458](https://github.com/godotengine/godot/pull/81458), [GH-88730](https://github.com/godotengine/godot/pull/88730)).
```